### DISCUSSION TITLE
When I select the options in multi-select Dropdown in ReactSelect, the scroller position is going to top instead of staying at the selected position

### DISCUSSION THREAD

**Discussion by Rutra-Selvi-Thavamani** [JedWatson/react-select#5575] [Q&A]
on selecting the options , scroller position should be consistent to the selected options

![image](https://user-images.githubusercontent.com/119167992/222948134-f67834b6-cf7c-4801-9c10-74863b96db56.png)

here is my code;
![image](https://user-images.githubusercontent.com/119167992/222948143-91c72292-f3d6-4cc3-9fd7-bb975fa01858.png)

**Comment by manjushsh** [comment]
Hi @Rutra-Selvi-Thavamani This is not default behavior in react-select I think. As you can [see here in docs example](https://react-select.com/home#animated-components).
